JoyboyCommunity/assets/svgs/svgComponents/Exit.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
+import Svg, { SvgProps, Rect, Path } from "react-native-svg"
+const Exit = (props: SvgProps) => (
```

Comment:
SVGs should be moved to [icons.tsx](https://github.com/keep-starknet-strange/joyboy/blob/5264c5680e32db71bf8b18f5d491d972925ab64f/JoyboyCommunity/src/assets/icons.tsx) file.